internal/controllers/image/tests/create-full/00-assert.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+resourceRefs:
+    - apiVersion: openstack.k-orc.cloud/v1alpha1
+      kind: Image
+      name: create-full
+      ref: image
+assertAll:
+    # Tags order is not consistent
+    - celExpr: "image.status.resource.tags.size() == 2"
```

Comment:
Nice👌

Do you want to additionally assert that the list contains the 2 expected tags? (Assuming CEL doesn't have a native way to do a non-order specific equality check on lists)